src/os/bluestore/BlueFS.cc

```diff
@@ -2208,6 +2209,22 @@ int64_t BlueFS::_read(
                  << std::hex << x_off << "~" << l << std::dec
                  << " of " << *p << dendl;
 	int r;
+	if (h->file->fnode.ino == 1) {
```

Comment:
wrap the code with a comment that this is a temporary stuff which to be eliminated from the final release?